pimd/pim_cmd.c

```diff
@@ -6861,6 +6878,28 @@ DEFUN (no_ip_pim_packets,
 	return CMD_SUCCESS;
 }
 
+DEFPY (igmp_group_watermark, igmp_group_watermark_cmd,
+       "[no$no] ip igmp watermark-warn [(10-60000)$limit]",
+       NO_STR
+       IP_STR
+       IGMP_STR
+       "Configure group limit for watermark warning\n"
+       "Group count to generate watermark warning\n")
+{
+	PIM_DECLVAR_CONTEXT(vrf, pim);
+	if (no)
+		limit = 0;
+	else {
+		if (!limit) {
+			vty_out(vty, "Command rejected: specify limit to warn\n");
```

Comment:
really should split the command, `ip igmp watermark-warn` shouldn't be available to the user